### ISSUE TITLE
Add Typescript definitions

### ISSUE THREAD

**Issue by 9ParsonsB** [kaisermann/svelte-css-vars#7]
Using this module in a TypeScript project with `"noImplicitAny": true` causes errors as there is no definitions file.
Please add a .d.ts file for this package. Here is my quick attempt at it that seems to resolve the issue:

```
declare module 'svelte-css-vars'
{
    export function update(new_props: any);
    function apply(node: HTMLElement, props: any);
    export default apply;
}
```

**Comment by kaisermann** [comment]
Hey, @9ParsonsB 👋 Thanks for the suggestion.  Released in v0.0.2 🎉

**Comment by 9ParsonsB** [comment]
:wave: Thanks! 🎉
